src/common/storage/src/operator.rs

```diff
@@ -93,6 +97,23 @@ pub fn init_fs_operator(cfg: &StorageFsConfig) -> Result<Operator> {
     Ok(Operator::new(builder.build()?).layer(LoggingLayer))
 }
 
+/// init_ftp_operator will init a opendal ftp operator.
+#[cfg(feature = "storage-ftp")]
+pub fn init_ftp_operator(cfg: &super::StorageFtpConfig) -> Result<Operator> {
+    let mut builder = ftp::Builder::default();
+
+    let bd = builder
+        .endpoint(cfg.endpoint.as_str())
+        .user(cfg.username.as_str())
+        .password(cfg.password.as_str())
+        .root(cfg.root.as_str());
+    if cfg.secure {
+        bd.enable_secure();
```

Comment:
Service FTP's API is weird. Either we provide `ftps://xxxx` or `enable_secure` instead of both of them. What happens if user inputs a `ftps://127.0.0.1` without `enable_secure`? Or a `ftp://127.0.0.1` with `enable_secure`?

We need to polish it here.

Reply:
Well, I got it. There is no `ftps` protocol provided by `service-ftp`, only `enable_secure`. I think we should:

- Remove `ftps` in opendal and databend
- Expose `enable_secure` to users directly

Reply:
I think we should let users use FTP like HTTP. Since the relationship between FTP and FTPS is really like that between HTTP and HTTPS.

Reply:
They are different.

`service-http` understand `http` and `https`, we pass them in the endpoint to HTTP client directly. But `service-ftp` doesn't. The FTP client accepts `192.168.1.1` and requires the user to set `enable_secure`.

Thus, we can't use `ftp` or `ftps` here. Instead, we use `ftp://exmaple.com` and always require the user to set `enable_secure`.

Reply:
Please revert PR https://github.com/datafuselabs/opendal/pull/704 and only use `ftp://`.

Reply:
> Well, I got it. There is no `ftps` protocol provided by `service-ftp`, only `enable_secure`.

The `enable_secure` means using `FTPS` actually. `FTPS` is TLS-enhanced `FTP`.

Reply:
> The `enable_secure` means using `FTPS` actually. `FTPS` is TLS-enhanced `FTP`.

Hey, we are confusing two layers of APIs.

First of all, opendal.

OpenDAL Service FTP provide only one API:

- `endpoint` like `127.0.0.1:9000` or `ftp.example.com`
- `enable_secure` to use tls or not.

Then, databend.

That's OK if you prefer to use `ftp` & `ftps`. In this hey, you must hide `enable_secure` from users.

- If user inputs `ftp://`, disable secure.
- If user inputs `ftps://`, enable secure.

Don't provide the same API across different layers.

Reply:
> Service FTP's API is weird. Either we provide ftps://xxxx or enable_secure instead of both of them. 

Got it, I'd rather remove the `enable_secure` option.

Reply:
> Got it, but I'd rather remove the `SECURE` option in `CONNECTION`.

OK, you are the boss of this feature. Go ahead!